scripts/utils/migrate_posix_kconfigs.py

```diff
@@ -0,0 +1,134 @@
+"""
+Utility script to migrate Zephyr-based projects to new POSIX Kconfig options
```

Comment:
Minor: It would be helpful to indicate here in which Zephyr version the rename happened. Both to guide users and so that later is clearer when this transitional script could be removed.

Reply:
Fixed!